org.osgi.test.common/src/main/java/org/osgi/test/common/service/ServiceConfiguration.java

```diff
@@ -17,18 +17,28 @@
 package org.osgi.test.common.service;
 
 import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.toList;
 import static org.osgi.test.common.filter.Filters.format;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.SortedMap;
+import java.util.function.Function;
+
 import org.osgi.framework.BundleContext;
 import org.osgi.framework.Filter;
+import org.osgi.framework.ServiceReference;
 import org.osgi.test.common.tracking.TrackServices;
```

Comment:
Is there still a good reason to leave TrackServices in its own package? At the minimum, we should move it into the common.services package. But perhaps we can go further and just merge it function directly into ServiceConfiguration?

Reply:
Done!